src/server/resources/venue-queries.yaml

```diff
@@ -1,4 +1,4 @@
-retrieveVenues: >
+readVenues: >
```

Comment:
`selectVenues`? I don't see a real reason to deviate from standard SQL verbs when useful.